assets/js/main.js

```diff
@@ -227,5 +271,61 @@ window.addEventListener('resize', () => {
     showDesktopElements()
 })
 
-// Logout listner
-document.querySelector('[js-header-auth-logout]').addEventListener('click', userLogout)
+window.addEventListener('load', function () {
+    // store tabs variable
+
+    const onTabClick = (event, pane, tabId) => {
+        event.preventDefault()
+
+        for (let i = 0; i < pane.children.length; i++) {
+            const tab = pane.children[i]
+            const active = tab.id == tabId
+
+            if (active) {
+                tab.classList.add('active')
+            } else {
+                tab.classList.remove('active')
+            }
+        }
+
+        const content = document.querySelector('#' + pane.id + '-content')
+        for (let i = 0; i < content.children.length; i++) {
+            const tabContent = content.children[i]
+            const active = tabContent.id + '-tab' == tabId
+
+            if (active) {
+                tabContent.classList.add('active')
+            } else {
+                tabContent.classList.remove('active')
+            }
+        }
+    }
+
+    const tabPanes = document.querySelectorAll('ul.nav-tabs')
+    tabPanes.forEach((pane) => {
+        pane.children[0].classList.add('active')
+        for (let i = 0; i < pane.children.length; i++) {
+            const tab = pane.children[i]
+            tab.addEventListener('click', (event) => onTabClick(event, pane, tab.id))
+        }
+    })
+
+    initCopyButtons()
+})
+
+if (typeof Storage !== 'undefined') {
+    const activeLanguage = localStorage.getItem('active_language')
+    if (activeLanguage) {
+        document.querySelectorAll('.persistLang-' + activeLanguage).forEach((element) => {
+            $('#' + element.id).tab('show')
+        })
+    }
+}
+function persistLang(language) {
```

Comment:
Use arrow function.

Reply:
We have our code styles, they have theirs. When you're adding code to our code base, you must make it fit with  our styles.